src/main/kotlin/com/sourcegraph/common/UpgradeToCodyProNotification.kt

```diff
@@ -85,12 +85,14 @@ private constructor(title: String, content: String, shouldShowUpgradeOption: Boo
     fun isCodyProJetbrains(project: Project): Boolean {
       val activeAccountType = CodyAuthenticationManager.instance.getActiveAccount(project)
       if (activeAccountType != null && activeAccountType.isDotcomAccount()) {
-        val server = CodyAgent.getServer(project)
-        if (server != null) {
-          val codyProFeatureFlag = server.evaluateFeatureFlag(GetFeatureFlag("CodyProJetBrains"))
-          return codyProFeatureFlag.get()!!
-        }
+        return CodyAgentService.getAgent(project)
+            .thenCompose { agent ->
+              agent.server.evaluateFeatureFlag(GetFeatureFlag("CodyProJetBrains"))
+            }
+            // TODO We should avoid using getNow if possible
```

Comment:
can we have an issue for this one? or does it make sense to just add `timeout(1, SECONDS)`?

Reply:
Timeout does not change anything there, because there is no quarantee if it will take 0, 2 or 10 sec.
I will create an issue.